qiita_db/support_files/patches/python_patches/40.py

```diff
@@ -0,0 +1,22 @@
+from random import SystemRandom
+from string import ascii_letters, digits
+
+from qiita_db.sql_connection import TRN
+
+pool = ascii_letters + digits
+with TRN:
+    # 2 and 3 are the ids of the 2 new software rows, the BIOM and
+    # target gene type plugins
+    for i in [2, 3]:
+        client_id = ''.join([SystemRandom().choice(pool) for _ in range(50)])
```

Comment:
This is to simulate the fact that there was no oauth key because this functionality was not in place? Why not just make NULL, or would that break something?